board.go

```diff
@@ -213,62 +213,67 @@ func PlaceFoodAutomatically(rand Rand, b *BoardState) error {
 func PlaceFoodFixed(rand Rand, b *BoardState) error {
 	centerCoord := Point{(b.Width - 1) / 2, (b.Height - 1) / 2}
 
-	// Place 1 food within exactly 2 moves of each snake, but never towards the center or in a corner
-	for i := 0; i < len(b.Snakes); i++ {
-		snakeHead := b.Snakes[i].Body[0]
-		possibleFoodLocations := []Point{
-			{snakeHead.X - 1, snakeHead.Y - 1},
-			{snakeHead.X - 1, snakeHead.Y + 1},
-			{snakeHead.X + 1, snakeHead.Y - 1},
-			{snakeHead.X + 1, snakeHead.Y + 1},
-		}
+	isSmallBoard := b.Width*b.Height <= BoardSizeSmall*BoardSizeSmall
```

Comment:
Wasn't totally sure about this. I felt like boards smaller than 7x7 should be included either though.